src/engine/sidechain/enginerecord.cpp

```diff
@@ -224,20 +224,23 @@ void EngineRecord::writeCueLine() {
                                     % 75);
 
     m_cueFile.write(QString("  TRACK %1 AUDIO\n")
-                    .arg((double)m_cueTrack, 2, 'f', 0, '0')
-                    .toUtf8());
+                            .arg((double)m_cueTrack, 2, 'f', 0, '0')
+                            .toUtf8());
 
     m_cueFile.write(QString("    TITLE \"%1\"\n")
-        .arg(m_pCurrentTrack->getTitle()).toUtf8());
+                            .arg(m_pCurrentTrack->getTitle())
+                            .toUtf8());
     m_cueFile.write(QString("    PERFORMER \"%1\"\n")
-        .arg(m_pCurrentTrack->getArtist()).toUtf8());
+                            .arg(m_pCurrentTrack->getArtist())
+                            .toUtf8());
 
     // Woefully inaccurate (at the seconds level anyways).
     // We'd need a signal fired state tracker
     // for the track detection code.
     m_cueFile.write(QString("    INDEX 01 %1:%2\n")
-                    .arg(getRecordedDurationStr())
-                    .arg((double)cueFrame, 2, 'f', 0, '0').toUtf8());
+                            .arg(getRecordedDurationStr())
+                            .arg((double)cueFrame, 2, 'f', 0, '0')
```

Comment:
If you touch that line anyway, please remove the C-style cast:

```suggestion
                            .arg(static_cast<double>(cueFrame), 2, 'f', 0, '0')
```